libbeat/docs/config-file-format.asciidoc

```diff
@@ -238,10 +238,10 @@ Example referencing `es.host` in `output.elasticsearch.hosts`:
 
 ["source","yaml",subs="attributes"]
 ------------------------------------------------------------------------------
-es.host: '$\{ES_HOST:localhost}'
+es.host: '${ES_HOST:localhost}'
 
 output.elasticsearch:
-  hosts: ['http://$\{es.host}:9200']
+  hosts: ['http://${es.host}:9200']
```

Comment:
@nik9000 Here's another change that results in errors in asciidoc, but fixes the asciidoctor content. The error is:

```
12:20:15 INFO:build_docs:asciidoc: WARNING: config-file-format.asciidoc: line 244: illegal system attribute
```

Reply:
I think the `{` is confusing asciidoc because those *look* like attributes to it. If you drop `subs="attributes"` it should just ignore them.